vote.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/gagliardetto/solana-go"
+)
+
+// IsSimpleVoteTransaction checks if a transaction is a simple vote transaction.
+// A simple vote transaction meets these conditions:
+// 1. has 1 or 2 signatures
+// 2. is legacy message (this is implicit in solana-go as it mainly handles legacy messages)
+// 3. has only one instruction
+// 4. which must be Vote instruction
+func IsSimpleVoteTransaction(tx *solana.Transaction) bool {
+    // Check signature count (condition 1)
+    if len(tx.Signatures) == 0 || len(tx.Signatures) > 2 {
+        return false
+    }
+
+    // Check instruction count (condition 3)
+    instructions := tx.Message.Instructions
+    if len(instructions) != 1 {
```

Comment:
can vote transactions also include compute budget? is that something validators do?

Reply:
I don't think I understand Solana data well enough to answer this question. @linuskendall do you know?